src/CommentBox/CommentBox.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import CommentList from './CommentList';
+import CommentForm from './CommentForm';
+import style from './style';
+
+class CommentBox extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { data: [] };
+    this.loadCommentsFromServer = this.loadCommentsFromServer.bind(this);
+    this.handleCommentSubmit = this.handleCommentSubmit.bind(this);
+    this.handleCommentDelete = this.handleCommentDelete.bind(this);
+    this.handleCommentUpdate = this.handleCommentUpdate.bind(this);
+    this.pollInterval = null;
+  }
+  loadCommentsFromServer() {
+    axios.get('http://localhost:3001/api/comments')
+      .then(res => {
+        this.setState({ data: res.data });
+      })
+  }
+  handleCommentSubmit(comment) {
+    let comments = this.state.data;
+    comment.id = Date.now();
+    let newComments = comments.concat([comment]);
+    this.setState({ data: newComments });
+    axios.post('http://localhost:3001/api/comments', comment)
+      .catch(err => {
+        console.error(err);
+        this.setState({ data: comments });
```

Comment:
Why do you rerender the component twice? Maybe will be better to rerender once when request was successful?